app/controllers/SaWrongCredentialsController.scala

```diff
@@ -48,16 +48,16 @@ class SaWrongCredentialsController @Inject()(
   private val authenticate: ActionBuilder[UserRequest, AnyContent] = authJourney.authWithSelfAssessment
 
   def ggSignInUrl: String = {
-    lazy val ggSignIn = s"${configDecorator.companyAuthHost}/${configDecorator.gg_web_context}"
+    lazy val ggSignIn = s"${configDecorator.basGatewayFrontendHost}/bas-gateway/sign-in"
```

Comment:
@vimalsuresh @anujakhare for a SA ambiguous user we ask then to sign in again using this page, /personal-account/self-assessment/sign-in-again 

This should send you to the correct sign-in route with the right continue_url set